service-statistics-unsd/src/main/java/fi/nls/oskari/control/statistics/plugins/unsd/UnsdIndicatorValuesFetcher.java

```diff
@@ -0,0 +1,81 @@
+package fi.nls.oskari.control.statistics.plugins.unsd;
+
+import fi.nls.oskari.control.statistics.data.IndicatorValue;
+import fi.nls.oskari.control.statistics.data.IndicatorValueFloat;
+import fi.nls.oskari.control.statistics.data.StatisticalIndicatorDataModel;
+import fi.nls.oskari.control.statistics.plugins.APIException;
+import fi.nls.oskari.control.statistics.plugins.unsd.parser.UnsdParser;
+import fi.nls.oskari.control.statistics.plugins.unsd.requests.UnsdRequest;
+import fi.nls.oskari.service.ServiceRuntimeException;
+import fi.nls.oskari.util.JSONHelper;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+public class UnsdIndicatorValuesFetcher {
+    private UnsdConfig config;
+
+    public void init(UnsdConfig config) {
```

Comment:
`UnsdIndicatorValuesFetcher` requires this init method to be called before it can be used. It really should be the constructor method instead.